docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: '3'
+
+services:
+
+  elasticsearch:
+    image: docker.elastic.co/elasticsearch/elasticsearch:5.3.0
+    container_name: audit_elasticsearch
+    ports:
+      - 9200:9200
+      - 9300:9300
+    environment:
+      # ES_JAVA_OPTS: "-Xmx256m -Xms256m"
```

Comment:
Updated compose file.  Was originally going for the most basic type of functional compose file possible.  However I've taken onboard the feedback amended it accordingly.

The compose file now spins up a 2 node cluster.

Updated the documentation also.